src/macros-by-example.md

```diff
@@ -71,6 +71,7 @@ syntax named by _designator_. Valid designators are:
 * `meta`: a [_MetaItem_], the contents of an attribute
 * `lifetime`: a [LIFETIME_TOKEN]
 * `vis`: a [_Visibility_] qualifier
+* `literal`: a [_LiteralPattern_] token. Examples: `4`, `-3`, `"String"`
```

Comment:
This doesn't quite capture that all literals can start with `-`.  I would do something like this for now:

```suggestion
* `literal`: matches `-`<sup>?</sup>[_LiteralExpression_]
```